vulndb/export.go

```diff
@@ -186,6 +186,10 @@ func (exp DataExporter) CSV(ctx context.Context, w io.Writer, header bool) error
 		})
 	}
 
+	if rows.Err() != nil {
+		return errors.Wrap(rows.Err(), "unable to read all rows from result set")
+	}
+
```

Comment:
This is unnecessary in this case. `errors.Wrap()` returns nil if the `err` is nil, so could just return `errors.Wrap(rows.Err(), "unable to...")`.